docs/guide.mdx

```diff
@@ -0,0 +1,432 @@
+
+```bash
+# default root directory: ~/.kwild
+$ kwild 
+
+(or)
+
+# Specify root directory explicitly
+$ kwild --root_dir "path/to/root/dir"
+
+(or)
+
+# If config files were not generated with kwil-admin, use quickstart mode
+$ kwild --autogen --root_dir "path/to/root/dir"
+```
+
+The above commands launches a private Kwil network comprising a single node, which hosts both a Kwil database and blockchain. Debug logs of the node's services during startup are shown below:
+
+```json title="Logging of Node Services Initialization"
+{"msg":"starting server..."}
+
+{"msg":"grpc server started","address":"localhost:50051"}
+
+{"msg":"http server started","address":"localhost:8080"}
+```
+
+The P2P and consensus engine's RPC service will have similar logs that display the effective listening addresses and the node's ID:
+
+```json
+"msg":"Add our address to book",
+"book":"~/.kwild/abci/config/addrbook.json",
+"addr":"b58257d0fd67c0757327ff16ec701990c24d5336@0.0.0.0:26656"
+
+"msg":"Starting RPC HTTPS server on 127.0.0.1:26657"
+"cert": "~/.kwild/rpc.cert", 
+"key": "~/.kwild/rpc.key"
+```
+
+Upon initializing all the services successfully, the Kwil node will begin mining blocks:
+
+```json title="Mining Blocks"
+// Block 1
+"msg":"finalizing commit of block",
+"module":"consensus",
+"height":1,
+"hash":"F1DFEC0D85C7D8D6FF827830DE0BFFA40A496B2388E295833231546614213767",
+"root":"C2CE859968C5AB7EC765BE39F1B12F27DFA1FBD1820C90501A4F13DE2581A4AC",
+"num_txs":4
+
+// Block 2
+"msg":"finalizing commit of block",
+"module":"consensus",
+"height":2,
+"hash":"A16C9BCC99D4DE0AF346BD8E261B750C7C4378D8A44F12B7D5BDF6F44F98A15A",
+"root":"CCBFD901930A79C2E9930A701608685381D29111ED4AA8355FB6F2654AD133C6",
+"num_txs":2
+```
+
+Once the Kwil node is up, users can start interacting with the Kwil database by using [`kwil-cli`](./kwil-cli/). The node operator can use [`kwil-admin`](./admin).
+
+### Verify gRPC Connectivity
+
+Users can `ping` the Kwil node to check the status of the Kwil node. If the node is active and online, it will reply with a `pong` message in response.
+
+```bash
+# Use --kwil-provider to specify the grpc listen addr of the Kwil node you wish to connect to.
+$ kwil-cli utils ping --kwil-provider "localhost:50051"
+pong
+```
+
+### Transactions
+
+Once the Kwil node is up, users can start issuing transactions with the help of the `kwil-cli` tool. Refer to the [database guide](./kwil-cli/databases) for full documentation on the CLI for available database operations.
+
+Below, we deploy a database using a transaction. We then call the blockchain using the returned transaction hash to verify that it was successful
+
+#### Deploy Database
+
+Database schema can be deployed on the Kwil nodes using `kwil-cli` as shown below. Refer to the [Kuneiform Guide](./category/kuneiform) to create your own database schema.
+:::tip
+You can write, compile, and deploy Kuneiform, as well as view sample Kuneiform files, online at https://ide.kwil.com.
```

Comment:
Probably replace "compile" with "parse".  The IDE uses "compile", but its actually not a correct description